tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py

```diff
@@ -71,6 +75,20 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
 
         # Validate pvc mounted on image registry pod
         validate_pvc_mount_on_registry_pod()
+        # Coverage for Bz 2128263
+        log.info(
+            f"Verifying whether alert {constants.ALERT_KUBEPERSISTENTVOLUMEINODESFILLINGUP} "
+            "has been triggered"
+        )
+        alerts = api.wait_for_alert(
+            name=constants.ALERT_KUBEPERSISTENTVOLUMEINODESFILLINGUP,
+            timeout=100,
+            sleep=1,
+        )
+        if len(alerts) > 0:
+            assert (
+                False
+            ), f"Failed: There should be no {constants.ALERT_KUBEPERSISTENTVOLUMEINODESFILLINGUP} alert"
```

Comment:
do we need to also check for kubelet_volume_stats_inodes in csi-cephfsplugin pod logs ?